tasks/libs/pipeline_data.py

```diff
@@ -130,3 +130,19 @@ def truncate_job_name(job_name, max_char_per_job=48):
     # We also want to avoid it being too long
     truncated_job_name = truncated_job_name[:max_char_per_job]
     return truncated_job_name
+
+
+# those jobs are `allow_failure: true` but still needs to be included
+# in fail reports
+jobs_allowed_to_fails_but_need_report = [re.compile(r'kitchen_test_security_agent.*')]
```

Comment:
As this is used only on your method below I would tend not to make it global, wdyt?

Reply:
I was trying to match what was done with `infra_failure_logs` in the same file